include/bitcoin/bitcoin/machine/rule_fork.hpp

```diff
@@ -70,6 +70,9 @@ enum rule_fork : uint32_t
     /// Prevent dummy value malleability (soft fork, feature).
     bip147_rule = 1u << 13,
 
+    /// Use litecoin blockchain validation (hard fork, litecoin).
+    litecoin = 1u << 14,
```

Comment:
Don't refer to specific coins, indicate each independent behavior change with an independent flag, e.g.:
```ini
[fork]
retarget_overflow_patch = false
script_proof_of_work = false
time_warp_patch =  false
```
and pass through code in this manner.

Reply:
We may of course encounter other scrypt pow algorithms, and other patches to these issues. There is a discussion in Bitcoin no regarding the time warp fix. I'd prefer to cross that bridge when we come to it rather than integrating coin names.

Reply:
I fully agree that it makes sense to avoid using coin names :+1: